library/sha1.c

```diff
@@ -290,21 +298,28 @@ void mbedtls_sha1_update( mbedtls_sha1_context *ctx, const unsigned char *input,
     if( left && ilen >= fill )
     {
         memcpy( (void *) (ctx->buffer + left), input, fill );
-        mbedtls_sha1_process( ctx, ctx->buffer );
```

Comment:
I mean replacing `memcpy( (void *) (ctx->buffer + left), input, fill );` by `memcpy( (void *) ( ctx->buffer + left ), input, fill );` and a few lines below as well.